applications/openshift/master/file_permissions_etcd_member/rule.yml

```diff
@@ -2,16 +2,17 @@ documentation_complete: true
 
 prodtype: ocp4
 
-title: 'Verify Permissions on the ETCD Member Pod Specification File'
+title: 'Verify Permissions on the etcd Member Pod Specification File'
 
 description: |-
-    {{{ describe_file_permissions(file="/etc/kubernetes/static-pod-resources/etcd-pod-.*/etcd-pod.yaml", perms="0644") }}}
+    {{{ describe_file_permissions(file="/etc/kubernetes/static-pod-resources/etcd-pod-*/etcd-pod.yaml", perms="0644") }}}
```

Comment:
What are the plans to move beyond 644? In security and compliance realm, you really don't check files that are system default file permissions, and the files themselves should really be 0600 for security.

Reply:
They don't contain any sensitive information, they're merely a way to tell the kubelet how to start the etcd pods. any sensitive information is stored as kubernetes secrets with appropriate permissions.

Reply:
also, CIS asks us to just check if they are not group or world writable, too.